quinn-proto/src/lib.rs

```diff
@@ -131,6 +131,13 @@ pub enum Directionality {
     Uni = 1,
 }
 
+impl Directionality {
+    #[doc(hidden)]
+    pub fn iter() -> impl Iterator<Item = Self> {
```

Comment:
I don't think this needs to be exported at all.

Reply:
Fixed.